build.proj

```diff
@@ -53,18 +53,17 @@
 
   <Target DependsOnTargets="$(CLITargets)" Name="BuildTheWholeCli"></Target>
 
+  <UsingTask TaskName="CompileTargets" AssemblyFile="$(CLIBuildDll)" />
```

Comment:
I believe UsingTask for CompileTargets and PackageTargets were removed in a previous checkin.